apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Resolvers/WholesaleResolvers.cs

```diff
@@ -24,14 +24,14 @@ public class WholesaleResolvers
     public async Task<string?> GetCreatedByUserNameAsync(
         [Parent] CalculationDto batch,
         UserBatchDataLoader dataLoader) =>
-        (await dataLoader.LoadAsync(batch.CreatedByUserId)).DisplayName;
+        (await dataLoader.LoadAsync(batch.CreatedByUserId))?.DisplayName;
 
     public async Task<IEnumerable<GridAreaDto>> GetGridAreasAsync(
         [Parent] CalculationDto batch,
         GridAreaByCodeBatchDataLoader dataLoader)
     {
         var gridAreas = await Task.WhenAll(batch.GridAreaCodes.Select(async c => await dataLoader.LoadAsync(c)));
-        return gridAreas.Where(g => g != null).OrderBy(g => g.Code);
+        return gridAreas.Where(g => g != null).Select(g => g!).OrderBy(g => g.Code);
```

Comment:
Also here, although the preceding `Where` prevents runtime errors at least. Does the compiler not infer the grid area to be non-null after the `Where` operator?

Reply:
Fixed.

Reply:
And no, Where() does not change nullability, so compiler will not see the change.